Providers/Pocket/PocketFeed.swift

```diff
@@ -35,53 +52,43 @@ class Pocket: FeatureFlaggable, URLCaching {
         return Bundle.main.object(forInfoDictionaryKey: PocketEnvAPIKey) as? String
     }()
 
-    // Fetch items from the global pocket feed
-    func globalFeed(items: Int = 2) -> Deferred<[PocketFeedStory]> {
-        if shouldUseMockData {
-            return getMockDataFeed(count: items)
-        } else {
-            return getGlobalFeed(items: items)
-        }
+    enum Error: Swift.Error {
```

Comment:
Again, I'm not sure what this `Swift.` business is. I've never seen it before so I'm super curious. Generally, the way we define errors should be the standard Swift way of inheritance `enum PocketFetchError: Error` or whatever. Is there a reason why we don't do that here?

If this is something new and cool, I'd love to learn. :)

Reply:
@electricRGB I also answered on the Result question. Hope it helps! In this case I want my error type to be named just `Error` because I can used it as `Pocket.Errror`, instead of coming with something like `Pocket.PocketError` which is repetitive. But if I'm using it as just plain `Error` the compiler get confused like what is that? Is it your error or the Swift error. So I have to give it some extra information.